docs/Maintainer-Guidelines.md

```diff
@@ -61,9 +61,12 @@ We now accept versioned formulae as long as they [meet the requirements](Version
 
 ### Merging, rebasing, cherry-picking
 
-Merging should be done in the `Homebrew/brew` repository to preserve history & GPG commit signing,
-and squash/merge via GitHub should be used for formulae where those formulae
-don't need bottles or the change does not require new bottles to be pulled.
+Merging should be done in the `Homebrew/brew` repository to preserve history & GPG commit signing.
```

Comment:
```suggestion
Merging should be done in the `Homebrew/brew` repository to preserve history and GPG commit signing.
```

This is the only other thing I think needs to be changed. Everything else looks good to me at this point.